structure.go

```diff
@@ -146,6 +147,8 @@ type MasterPlaylist struct {
 	CypherVersion string // non-standard tag for Widevine (see also WV struct)
 	buf           bytes.Buffer
 	ver           uint8
+	lock          *sync.Mutex
+	dirty         bool
```

Comment:
please add a comment indicating what elements this lock is protecting

Reply:
fixed